server/courselist.rb

```diff
@@ -6,7 +6,7 @@
 ### This version will also server up the HTML page if no
 ### specific page is requested.
 
-### Configuration will be read in from /usr/local/studentdashboard/studentdashboard.yml if available
+### Configuration will be read in from /usr/local/ctools/app/ctools/tl/home/studentdashboard.yml file if available
```

Comment:
Looks like the path is hard-coded in this code? At minimum, I'd suggest not duplicating the file path in this comment, but ideally it shouldn't be in this file at all -- studentdashboard.yml seems better than hard-coding in a ruby file.